diff --git a/node_modules/open/index.js b/node_modules/open/index.js
index 14d15cf..f8f3e4d 100644
--- a/node_modules/open/index.js
+++ b/node_modules/open/index.js
@@ -3,7 +3,8 @@ import {Buffer} from 'node:buffer';
 import path from 'node:path';
 import {fileURLToPath} from 'node:url';
 import childProcess from 'node:child_process';
-import fs, {constants as fsConstants} from 'node:fs/promises';
+import fs from 'node:fs/promises';
+import { constants as fsConstants } from 'fs';
 import isWsl from 'is-wsl';
 import defineLazyProperty from 'define-lazy-prop';
 import defaultBrowser from 'default-browser';
